src/components/Compare/CompareTable.tsx

```diff
@@ -86,7 +87,7 @@ const CompareTable = (props: {
   const getPopulation = (location: SummaryForCompare) =>
     location?.locationInfo?.population;
   const getMetricValue = (location: any) =>
-    location.metricsInfo.metrics[sorter].value;
+    location?.metricsInfo?.metrics[sorter]?.value;
```

Comment:
I don't think this change is needed? `location.metricsInfo.metrics[sorter].value` is always defined (either as a number or null)